### ISSUE TITLE
Some other codes of conduct to consider drawing from as needed

### ISSUE THREAD

**Issue by naupaka** [alxsimon/ecoevo.social-docs#3]
A short but by no means complete list of ideas:

* https://www.esa.org/events/code-of-conduct-for-esa-events/
* https://www.agu.org/Plan-for-a-Meeting/AGUMeetings/Meetings-Resources/Meetings-code-of-conduct
* https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html

**Comment by alxsimon** [comment]
Yes good to remind me of that, I initially planned to go see code of conducts of societies.

**Comment by alxsimon** [comment]
- https://www.evolutionmeetings.org/safe-evolution.html

**Comment by alxsimon** [comment]
Contributor covenant is interesting
https://www.contributor-covenant.org/version/2/1/code_of_conduct/

**Comment by playmobilmeister** [comment]
Regarding CWs and Sensitive content, I suggest that I translate our section from tooot.im. Then you can adapt it to ecoevo's special circumstances. It looks more or less like this:

## Intro
Discusses sensitivity in general

## Options for individuals
lets people know about what they can can do about annoyances, whether they be the CW buttonss themselves or the sensitive content, as well as filtration using tags

### Concept: Sensitivity ∝ ⨍(exposure level)

more details to come, but it ends with a table like this, which was our goal in allowing people to post with as few CWs or 'sensitive'-marked images while keeping the local feed free of acidental exposure to the benefit of the more sensitive members of the community

>Contents that must be warned against by hiding under an email address or media as sensitive in public publications >(including unlisted) include:
>
>❌ At this level of privacy, such content must be defined as sensitive if it is graphic, and hidden under a content warning if it is textual
>
>✅ At this level of privacy, there is no obligation to set such content as sensitive or hide it under a content warning
> 
|| public 🌏 | unlisted 🔓 | In followers only 🔒 |
| --- | --- | --- | --- |
| Political discussions ❌ | ✅ | ✅ |
| Animations or videos that flicker or change quickly | ❌ | ❌ | ❌ |
| Nudity, pornographic or highly sexual content, including textual pornographic content ❌ | ❌ | ✅ |
| Any content that shows violence (including graphic representations of dead people and dead animals, and of armed people) or talks about the prohibited content on the server | ❌ | ❌ | ✅ |
| Images of food that includes meat | ❌ | ❌ | ✅ |

Ideally, this should be done after some deliberation among members of the admin group as to what the actual principles and rules should be. This is only given here to present a structure of a conduct section that we created gradually, following a long process in our own community. If you like this sort of stuff, I can translate the whole section (this time not poorly) and other sections according to ecoevo.social's needs.

**Comment by alxsimon** [comment]
Thanks @playmobilmeister, the table looks like a good idea to adapt.
I won't really have time to spearhead the code of conduct topic in the following weeks though.
But if other admins/moderators feel they want to start organizing a discussion around a Code of Conduct this would be great.
